src/helpers/minifier.js

```diff
@@ -18,10 +17,7 @@ function minify(minOperation, opts, cb) {
 
   gulp.src(src)
     .pipe(minOperation(options))
-    .on('error', (err) => {
-      log.error(`Error has occured: ${err}`);
-      cb();
-    })
+    .on('error', cb)
```

Comment:
Should not we log here any information? Something like: 'Minification is broken'?